auction-app-backend/src/main/java/com/atlantbh/auctionappbackend/security/JwtConfig.java

```diff
@@ -0,0 +1,49 @@
+package com.atlantbh.auctionappbackend.security;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "application.jwt")
+public class JwtConfig {
+
+    private String secretKey;
+    private String tokenPrefix;
+    private Integer tokenExpAfterMin;
+    private Integer refreshTokenExpAfterMin;
+
+    public JwtConfig() {
+    }
```

Comment:
Since this is the only contractor in this class, we do not need to specify them (by default, each class has empty constructor).

Reply:
Sure, I'll remove it and keep it in mind.